pkg/models/dashboards.go

```diff
@@ -346,6 +346,12 @@ type SaveDashboardCommand struct {
 	Result *Dashboard
 }
 
+type TrimDashboardCommand struct {
+	Dashboard *simplejson.Json `json:"dashboard" binding:"Required"`
+	Meta      *simplejson.Json `json:"meta"`
```

Comment:
Not used currently, remove? Or maybe useful?

Reply:
it is used by TrimDashboard, since meta is not built but directly take from input and returned in output